desktop/hardened-desktop/enhanced-container-isolation/faq.md

```diff
@@ -1,79 +1,70 @@
 ---
-title: FAQs and known issues
+title: FAQs 
 description: FAQ for Enhanced Container Isolation
-keywords: enhanced container isolation, security, faq, sysbox, known issues, Docker Desktop
+keywords: enhanced container isolation, security, faq, sysbox, Docker Desktop
 toc_max: 2
 ---
 
-<ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" data-target="#tab3">FAQs</a></li>
-  <li><a data-toggle="tab" data-target="#tab4">Limitations and Known Issues</a></li>
-</ul>
-<div class="tab-content">
-<div id="tab3" class="tab-pane fade in active" markdown="1">
+### Do I need to change the way I use Docker when Enhanced Container Isolation is switched on?
 
-#### Do I need to change the way I use Docker when Enhanced Container Isolation is enabled?
+No, you can continue to use Docker as usual. 
 
-No, you can continue to use Docker as usual. Enhanced Container Isolation will be mostly transparent to you.
-
-#### Do all container workloads work well with Enhanced Container Isolation?
+### Do all container workloads work well with Enhanced Container Isolation?
```

Comment:
@aevesdocker Actually, as you're changing them anyway, technically these should all be H2s